python/src/space/core/runners.py

```diff
@@ -191,8 +202,17 @@ class LocalRunner(BaseReadWriteRunner):
   def read(self,
            filter_: Optional[pc.Expression] = None,
            fields: Optional[List[str]] = None,
+           tag: Optional[str] = None,
```

Comment:
Could we combine tag and snapshot_id into one field, maybe called `version`, with type Union[str, int]? If it is a string, it is a tag, if it is an int, it is a snapshot ID.

Reply:
Done